views/register.pug

```diff
@@ -10,14 +10,14 @@ block content
       div(class='register-form')
         form(action='/register', method='post' )
           input(type='hidden', name='_csrf', value=csrfToken)
-          +field('First Name', 'firstName', firstName)
-          +field('Last Name', 'lastName', lastName)
-          +field('Username', 'username', username)
-          +field('Email', 'email', email)
-          +field('Password', 'password', null, 'password')
-          +field('Confirm Password', 'confirmPassword', null, 'password')
+          +field('First Name', 'First Name', firstName)
```

Comment:
I think this might break the register page, because the second arg is supposed to pass the value as this variable name

Reply:
To change the placeholder, change `placeholder=placeholder` on line 17 in utils.pug